base/linalg/symmetric.jl

```diff
@@ -57,10 +57,29 @@ ctranspose(A::Hermitian) = A
 trace(A::Hermitian) = real(trace(A.data))
 
 #tril/triu
-tril(A::Hermitian,k::Integer=0) = tril(A.data,k)
-triu(A::Hermitian,k::Integer=0) = triu(A.data,k)
-tril(A::Symmetric,k::Integer=0) = tril(A.data,k)
-triu(A::Symmetric,k::Integer=0) = triu(A.data,k)
+function tril(A::Union{Hermitian,Symmetric}, k::Integer=0)
```

Comment:
Ack, I was wrong, while the earlier implementations were identical for Hermitian vs Symmetric, they actually shouldn't be. We can have Symmetric for a complex element type in which case these should be using `.'` instead of `'`. But for Hermitian they should use `'`.